README.md

```diff
@@ -173,15 +177,14 @@ you would pandas.
 <img align="right" style="display:inline;" height="350" width="300" src="https://github.com/modin-project/modin/blob/master/docs/img/read_csv_benchmark.png?raw=true"></a>
 
 The `modin.pandas` DataFrame is an extremely light-weight parallel DataFrame. Modin 
-transparently distributes the data and computation so that all you need to do is
-continue using the pandas API as you were before installing Modin. Unlike other parallel
-DataFrame systems, Modin is an extremely light-weight, robust DataFrame. Because it is
-so light-weight, Modin provides speed-ups of up to 4x on a laptop with 4 physical cores.
+transparently distributes the data and computation so that you can continue using the same pandas API
+while being able to work with more data faster. Because it is so light-weight, 
+Modin provides speed-ups of up to 4x on a laptop with 4 physical cores.
 
 In pandas, you are only able to use one core at a time when you are doing computation of
-any kind. With Modin, you are able to use all of the CPU cores on your machine. Even in
-`read_csv`, we see large gains by efficiently distributing the work across your entire
-machine.
+any kind. With Modin, you are able to use all of the CPU cores on your machine. Even in a
+traditionally synchronous task like `read_csv`, we see large gains by efficiently 
```

Comment:
suggestion - replace large gains with something like speedups.

Even with a typically synchronous task like `read_csv`, we see large speedups by distributing the work across your entire machine.